app/forms/signin/code_validations.rb

```diff
@@ -0,0 +1,26 @@
+module Signin
+  module CodeValidations
```

Comment:
This refactor could also be done in [SharedFiles CreateForm](https://github.com/inloco/incognia/blob/master/app/forms/shared_files/create_form.rb) for a part of the file validations when [[Admin] Create Shared File](https://www.notion.so/Admin-Create-Shared-File-2f5710be8fbd4c969db1924a6614c040) is prioritized.
Very well done here 👏🏼

Reply:
I'm not a very big fan, but it felt okay here. :grin:

Reply:
I liked it 👀